pymodbus/pdu/register_read_message.py

```diff
@@ -109,6 +109,14 @@ def getRegister(self, index):
         """
         return self.registers[index]
 
+    @property
+    def truncated(self) -> bool:
+        """Check if the response has been truncated."""
```

Comment:
You are missing a lot of requests that also have length

if implemented it must be implemented for all requests !

Reply:
Yeah, I just started with this one, still need to go through the others and add truncation properties.